Loop/Managers/LoopDataManager.swift

```diff
@@ -617,9 +617,11 @@ extension LoopDataManager {
     /// - Parameters:
     ///   - events: The pump events to add
     ///   - completion: A closure called once upon completion
+    ///   - lastReconciliation: The date that pump events were most recently reconciled against recorded pump history. Pump events are assumed to be reflective of delivery up until this point in time. If reservoir values are recorded after this time, they may be used to supplement event based delivery.
+    ///   - parameter updateLastReconciliation: Whether or not the last reconciliation time should be updated with the new value.
```

Comment:
Same comment here as on the LoopKit PR; the reason this parameter is necessary isn't clear to me.